app/models/concerns/project_media_getters.rb

```diff
@@ -213,4 +213,10 @@ def team_avatar
   def fact_check
     self.claim_description&.fact_check
   end
+
+  def explainers_title
+    # Get the title for all explainer assinged to the item
+    titles = Explainer.joins(:explainer_items).where('explainer_items.project_media_id = ?', self.id).map(&:title).join(' ')
```

Comment:
I know that in terms of indexing it doesn't make a huge difference, but for clarity, I suggest to use something else to concatenate the titles, since a title can contain a space. Newline, maybe?

Reply:
Sorry @caiosba, what’s the other suggestion? Can you give me an example?

Reply:
For example: `join("\n")`.